src/snyk/common/languageServer/languageServer.ts

```diff
@@ -33,8 +36,10 @@ export class LanguageServer implements ILanguageServer {
   async start(): Promise<void> {
     // TODO remove feature flag when ready
     if (!this.configuration.getPreviewFeatures().lsAuthenticate) {
+      await this.contextService.setContext(SNYK_CONTEXT.PREVIEW_LS_AUTH, false);
       return Promise.resolve(undefined);
```

Comment:
You don't need to pass `undefined` in here,  it's implicit in `Promise.resolve()`